### DISCUSSION TITLE
error getting repository remote digest... x509: certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by samothij** [artifacthub/hub#2168] [General]
I am running a local instance of artifacthub and connecting to a helm repo on a local server with a self signed certificate.
Is there a parameter that would let me bypass this x509 error?

![x509](https://user-images.githubusercontent.com/75305677/181448741-c575d988-050d-4256-ae16-bd69b84984c4.jpg)

**Comment by tegioz** [comment]
Hi @samothij 👋 

No, there isn't any option at the moment to skip certificates verification.

**Reply by samothij** [reply]
Gotcha.  Thanks much!

In a slightly different use case, I passed a value, TRIVY_INSECURE=true,  to scanner (internal/scanner/scanner.go), and got trivy to skip certificate verification.

Having an option to skip cert verification across the board would be awesome for us running local instances on private repos.

**Reply by tegioz** [reply]
No worries! Yes, that may be handy, thanks 👍
